tensorflow/python/autograph/operators/py_builtins_test.py

```diff
@@ -123,6 +123,43 @@ def test_len(self):
       tl = py_builtins.len_(data_structures.tf_tensor_list_new([3, 4, 5]))
       self.assertEqual(self.evaluate(tl), 3)
 
+  def test_len_dataset(self):
```

Comment:
Sorry, I missed an issue that is blocking internal tests - these can also run with eager mode off. To fix that, could you modify the checks as follows:

`self.assertEqual(self.evaluate(py_builtins.len_(dataset)), 3)`
`self.assertEqual(self.evaluate(test_fn()), 3)`

and so on

That way they'll be treated consistently in both eager and graph modes.

Reply:
@mdanatg Thanks. The PR has been updated.